addons/account/tests/test_account_move_duplicate.py

```diff
@@ -87,8 +87,9 @@ def test_in_invoice_multiple_duplicate_reference_constrains(self):
         invoices.ref = invoice_1.ref
 
         # test constrains: batch without any previous posted invoice
-        with self.assertRaises(RedirectWarning):
+        with self.assertRaises(RedirectWarning) as cm:
             (invoice_1 + invoice_2 + invoice_3).action_post()
+        self.assertEqual(cm.exception.args[1]['domain'], [('id', 'in', invoices.ids)])
```

Comment:
Your runbot is red because:
`AssertionError: Lists differ: [('id', 'in', [548, 549, 543])] != [('id', 'in', [543, 548, 549])]`
You could sort the ids / browse them and compare recordset / remove this line. I'll let you pick :)

Reply:
We should also add a comment above explaining this is the domain of the redirectWarning because it is a bit unclear at first sight.